src/container/srv_target.c

```diff
@@ -1298,7 +1298,9 @@ ds_cont_tgt_destroy_handler(crt_rpc_t *rpc)
 	D_DEBUG(DB_MD, DF_CONT": handling rpc %p\n",
 		DP_CONT(in->tdi_pool_uuid, in->tdi_uuid), rpc);
 
-	rc = ds_cont_tgt_destroy(in->tdi_pool_uuid, in->tdi_uuid);
+	if (!DAOS_FAIL_CHECK(DAOS_CHK_CONT_ORPHAN))
+		rc = ds_cont_tgt_destroy(in->tdi_pool_uuid, in->tdi_uuid);
```

Comment:
I think this is all we really need now, right? I think we can probably just use the `daos_debug_set_params` utility for this. I could make dmg shell out to that utility because it's not very nice to use, though.

Reply:
It am not sure whether it is convenient for @shimizukko to directly use the C API in his CR demo scripts or we have some utils named **daos_debug_set_params**?